src/main/java/frc/robot/Robot.java

```diff
@@ -1,13 +1,32 @@
 package frc.robot;
 
 import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import frc.robot.commands.IntakeIn;
+import frc.robot.commands.IntakeOut;
+import frc.robot.subsystems.Intake;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import frc.robot.commands.RotateShooter;
+import frc.robot.subsystems.Shooter;
 
 public class Robot extends TimedRobot {
+    private Shooter shooter;
+    private XboxController xboxController;
+    private Intake intake;
 
     @Override
     public void robotInit() {
+        intake = new Intake();
+        xboxController = new XboxController(0);
+        new JoystickButton(xboxController, XboxController.Button.kY.value).onTrue(new IntakeIn(intake));
+        new JoystickButton(xboxController, XboxController.Button.kX.value).whileTrue(new IntakeOut(intake));
 
+        this.shooter = new Shooter();
```

Comment:
initialize all of the systems together.  
```
this.intake = new Intake();
this.shooter = new Shooter();
```

Reply:
additionally: initialize systems before configuring buttons and commands.

This will help to avoid problems caused by initialization order

Reply:
I fixed it.